scipy/interpolate/_cubic.py

```diff
@@ -430,18 +471,56 @@ class CubicSpline(PPoly):
     >>> plt.legend(loc='lower left', ncol=2)
     >>> plt.show()
 
+    In the second example, the unit circle is interpolated with a spline. A
+    periodic boundary condition is used. You can see that the first derivative
+    values, ds/dx=0, ds/dy=1 at the periodic point (1, 0) are correctly
+    computed. Note that a circle cannot be exactly represented by a cubic
+    spline. To increase precision, more breakpoints would be required.
+
+    >>> theta = 2 * np.pi * np.linspace(0, 1, 5)
+    >>> y = np.c_[np.cos(theta), np.sin(theta)]
+    >>> cs = CubicSpline(theta, y, bc_type='periodic')
+    >>> print("ds/dx={:.1f} ds/dy={:.1f}".format(cs(0, 1)[0], cs(0, 1)[1]))
+    ds/dx=0.0 ds/dy=1.0
+    >>> xs = 2 * np.pi * np.linspace(0, 1, 100)
+    >>> plt.figure(figsize=(6.5, 4))
+    >>> plt.plot(np.cos(xs), np.sin(xs), label='true')
+    >>> plt.plot(y[:,0], y[:,1], 'o', label='breakpoints')
+    >>> plt.plot(cs(xs)[:,0], cs(xs)[:,1], label='spline')
+    >>> plt.axes().set_aspect('equal')
+    >>> plt.legend(loc='center')
+    >>> plt.show()
```

Comment:
This is really really minor an non-binding: While you're at it, it'd be good to tweak the example. At least spaces after commas please.